changelog/50597.added

```diff
@@ -0,0 +1,4 @@
+Adds arguments `success_stdout` and `success_stderr` to execution modules and states
+that use the underlying `cmd._run` functionality. These arguments allow the user
+to override the return code of the command depending on the contents of stdout
+or stderr.

```

Comment:
**suggestion**: changelog description should be as concise as possible - it will be linked back to the original issue for those who need more follow up.

This might be:

> Added `success_stdout` and `success_stderr` arguments to `cmd.run`, to override default return code behavior.

Reply:
Done